src/base/bittorrent/session.cpp

```diff
@@ -2432,8 +2454,24 @@ const QStringList Session::getListeningIPs()
 
     if (ifaceName.isEmpty()) {
         if (!ifaceAddr.isEmpty())
-            IPs.append(ifaceAddr);
-        else
+        {
+            if (!ifaceAddrPattern.isEmpty() && !addressPatternMatches(ifaceAddr, ifaceAddrPattern))
+            {
+                logger->addMessage(tr("Configured network interface address %1 doesn't match the given pattern.", "Configured network interface address 124.5.1568.1 doesn't match the given pattern.").arg(ifaceAddr), Log::CRITICAL);
```

Comment:
Typo: 124.5.1568.1

Reply:
Whoops, it's a silly copy/paste error (I've copied it from the log message for the invalid address).